plugin/src/main/java/org/wildfly/plugin/provision/PackageServerMojo.java

```diff
@@ -160,6 +160,13 @@ public class PackageServerMojo extends AbstractProvisionServerMojo {
     @Parameter(defaultValue = "false", property = PropertyNames.SKIP_PACKAGE)
     private boolean skip;
 
+    /**
+     * Set to {@code true} if you want to skip deploying the deployung in the provisioned server.
```

Comment:
Is "deployung" supposed to be "deployment"?

Reply:
typing too fast! I've updated the javadoc, thanks